egov-indexer/src/main/java/org/egov/infra/indexer/producer/IndexerProducer.java

```diff
@@ -12,7 +12,7 @@
 	public static final Logger logger = LoggerFactory.getLogger(IndexerProducer.class);
 
 	@Autowired
-    private CustomKafkaTemplate<String, Object> kafkaTemplate;
+    private KafkaTemplate<String, Object> kafkaTemplate;
```

Comment:
Any reason to move to `KafkaTemplate`?

Reply:
These changes were done by Abhilash for UAT. Looks like they wanted to use different version of kafka than tracer on old springboot  for using kafka connect.